### PR TITLE
Add development.yml

### PR DESCRIPTION
**Pull request by Blackjacx**
I maintain this project. I think it would be nice to get some help here and there 👍

### CODE REVIEW

File: _data/projects/development.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  - swift 
+  - ios 
+  - work 
+  - code
+  - hiring 
+  - features 
+  - guides
+upforgrabs:
+  name: help wanted
+  link: https://github.com/blackjacx/development/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22
```
Comment (shiftkey) [inline _data/projects/development.yml]:
```suggestion
  link: https://github.com/Blackjacx/Development/labels/help%20wanted
```

File: _data/projects/development.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+name: Development
```
Comment (shiftkey) [inline _data/projects/development.yml:1]: 🤔 is there a more specific name that you could use here?

**Comment by shiftkey** [comment]
Closing this out due to inactivity. Feel free to resubmit based on the feedback provided...

**Comment by Blackjacx** [comment]
I guess it is ok when this repo is ot included 👍 Thanks for your support!

**Pull request closed without merging**